contracts/external/WETH9.sol

```diff
@@ -1,90 +0,0 @@
-/**
```

Comment:
Removing this contract causes tests to break. Either remove the failing tests as well, or add this contract back.